recipe_scrapers/rezeptwelt.py

```diff
@@ -9,19 +25,69 @@ def host(cls):
         return "rezeptwelt.de"
 
     def site_name(self):
-        raise StaticValueException(return_value="Rezeptwelt")
+        return "Thermomix Rezeptwelt"
 
     def author(self):
-        return normalize_string(self.soup.find("span", {"id": "viewRecipeAuthor"}).text)
+        tag = self.soup.find("div", itemprop="author")
+        if tag:
+            return normalize_string(tag.get_text())
+        tag = self.soup.find("span", {"id": "viewRecipeAuthor"})
+        return normalize_string(tag.get_text())
```

Comment:
Some observations here:

* The retrieval from an `itemprop="author"` attribute is essentially `schema.org` metadata retrieval; we have an [existing helper method](https://github.com/hhursev/recipe-scrapers/blob/db6c82f24be5672f1389da53f4b3ccc9ecf0d176/recipe_scrapers/_schemaorg.py#L126) to implement that, so let's re-use them here.
* The information contained in the `viewRecipeAuthor` element seems more-specific than the schema metadata, which is sometimes generic.  So let's prefer `viewRecipeAuthor` when mentioned.

What this leads me to when adapting the code locally is:

```suggestion
        name_from_schema = self.schema.author()

        name_from_hyperlink = None
        tag = self.soup.find("span", {"id": "viewRecipeAuthor"})
        if tag:
            name_from_hyperlink = tag.get_text()

        return normalize_string(name_from_hyperlink or name_from_schema)
```

Note: the word `von` in some of the test data seems redundant, so we can remove that (these changes affect that).